docs/api/resource-reaper.md

```diff
@@ -0,0 +1,29 @@
+# Resource Reaper
+
+Testcontainers automatically assigns a Resource Reaper session id to each Docker resource. After the tests are finished — whether they are successful or not — [Ryuk][moby-ryuk] will take care of the remaining Docker resources and removes them. You can change the Resource Reaper session and group Docker resources together. Right now, only Linux containers are supported.
```

Comment:
Why would a user group Docker resources under a specific Resource Reaper id? What is the advantage of using the same id across all resources?

Reply:
Not sure about the use case. As I said, it is supported through the builder implementation. It was added by a contribution from Codecentric. Dunno - if they have a customer or use it somewhere. Removing it would be a breaking change.

Reply:
We don't have to remove the functionality right now, but having the docs like this would likely make users wonder, what are the implicit effects of grouping Docker resources. And then they might assume certain effects, that actually aren't present. So being explicit could help the docs here.